docker/kubernetes-tentacle/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
```

Comment:
Once we get things running, I'd love to see if `6.0-alpine` works...

Reply:
yeah, it probably will, the only trick is that it doesn't have bash installed which the install script needs 😁

Reply:
or, as @EddyMoulton said, the new chiseled images for ubuntu while could be good too